File: api/src/main/java/com/hirrao/javaee/service/HeartRateService.java
package com.hirrao.javaee.service;

import com.hirrao.javaee.entity.HeartRate;
import org.springframework.stereotype.Service;
import java.util.List;

@Service
public interface HeartRateService {
    List<HeartRate> searchHeartRateById(Long uid);
    void insertHeartRate(long hrlId, long uid, float heartRate,String recordTime);
}

File: api/src/main/kotlin/com/hirrao/javaee/service/HeartRateService.kt
package com.hirrao.javaee.service

import com.hirrao.javaee.entity.HeartRate
import org.springframework.stereotype.Service

@Service
interface HeartRateService {
    fun searchHeartRateById(uid: Long?): List<HeartRate?>?
    fun insertHeartRate(hrlId: Long, uid: Long, heartRate: Float, recordTime: String?)
}
